src/applications/ivc-champva/10-7959C/config/form.js

```diff
@@ -126,13 +125,26 @@ const formConfig = {
   subTitle: 'CHAMPVA Other Health Insurance Certification (VA Form 10-7959c)',
   defaultDefinitions: {},
   chapters: {
+    formSignature: {
+      title: 'Signer information',
+      pages: {
+        formSignature: {
+          path: 'form-signature',
+          title: 'Form signature',
+          ...formSignatureSchema,
+        },
+      },
+    },
```

Comment:
Shifted this screen from the last position in the form flow to the first position.